### DISCUSSION TITLE
SubPatch Instances

### DISCUSSION THREAD

**Discussion by constructor** [cables-gl/cables_docs#546] [Feature Requests] [answered]
I would be fantastic to have SubPatches as more of a blueprint with instances, rather than simply copies of bundled nodes ([that don't even copy properly](https://github.com/cables-gl/cables_docs/issues/274)). As things stand, subpatches are a great idea, but they have real issues.
Having SubPatches that work in a class/instance way would unleash great power and potential.

**Comment by pandrr** [comment]
yeah we are working on that

**Reply by constructor** [reply]
That is great news. Having the ability to create real, modular functionality that can be instanced will enable the creation of MUCH more complex stuff. Thanks for all you guys do.

**Reply by pandrr** [reply]
totally we wish we had this earlier, but its one of the big features of the next release hopefully. also i will show my progress in the weekly meetups!

**Reply by steam0r** [reply, marked as answer]
so, "next release" meant "this release". try it out!

https://blog.cables.gl/2024/02/01/february-2024-release/

**Comment by constructor** [comment]
Great work to all those involved. I have been away a while, but will definitely now be looking at this!
